docs-generation/src/examples.md

```diff
@@ -0,0 +1,365 @@
+# Usage examples
+
+All of the examples are available [in the sources](https://github.com/softwaremill/sttp/blob/master/examples/src/main/scala/sttp/client/examples) in runnable form.
+
+## POST a form using the synchronous backend
+
+Required dependencies:
+
+```scala            
+libraryDependencies ++= List("com.softwaremill.sttp.client" %% "core" % "2.2.0")
+```
+
+Example code:
+
+```scala mdoc:compile-only
```

Comment:
I only added mdoc annotation here, just to show that it works. You can try it by yourself, just break this code and try to regenerate docs